README.md

```diff
@@ -98,6 +98,8 @@ This directory is also included in `.gitignore` not to be shown as pending chang
 
 Since version 3.0.0 of this SDK, Sentry version >= v20.6.0 is required. This only applies to on-premise Sentry, if you are using [sentry.io](http://sentry.io/) no action is needed.
```

Comment:
```suggestion
Since version 3.0.0 of this SDK, Sentry version >= v20.6.0 is required. This only applies to self-hosted Sentry, if you are using [sentry.io](http://sentry.io/) no action is needed.
```